app/services/import_service.rb

```diff
@@ -124,6 +126,28 @@ def import_bookmarks!
     end
   end
 
+  def import_lists!
+    parse_import_data!(['List name', 'Account address'])
+    items = @data.take(ROWS_PROCESSING_LIMIT).map { |row| [row['List name'], row['Account address']] }
+    list_titles = items.map { |row| row[0] }.uniq
+
+    if @import.overwrite?
+      List.where(title: list_titles, account: @account).destroy_all
+    end
+
+    items.each do |listname, accountname|
+      list = @account.lists.find_or_create_by!(account: @account, title: listname)
+      username, domain = accountname.split('@')
+
+      acct = domain == Rails.configuration.x.local_domain ? Account.find_local(username) : Account.find_remote(username, domain)
```

Comment:
`acct` usually refers to the `username@domain` string. Use `TagManager.instance.local_domain?` to check if the domain is local.